contributors/design-proposals/customresources-subresources.md

```diff
@@ -131,17 +90,32 @@ type ScaleStatus struct {
 The `SubResources` field in `CustomResourceDefinitionSpec` will be gated under the `CustomResourceSubResources` alpha feature gate.
 If the gate is not open, the value of the new field within `CustomResourceDefinitionSpec` is dropped on creation and updates of CRDs.
 
+### Scale type
+
+The `Scale` object is the payload sent over the wire for `/scale`. The [polymorphic `Scale` type](https://github.com/kubernetes/kubernetes/pull/53743) is used for the `Scale` object.
+
 ## Semantics
 
 ### Validation Behavior
 
 #### Status
 
 The status endpoint of a CustomResource receives a full CR object. Changes outside of the `.status` subpath are ignored.
-For validation everything outside of the `.status` subpath can be reset to the existing values of the CustomResource. Then the JSON Schema presented in the CRD is validated against the whole object.
+For validation, the JSON Schema present in the CRD is validated only against the `.status` subpath.
+
+To validate only against the schema for the `.status` subpath, `oneOf` and `anyOf` constructs are not allowed within the root of the schema
```

Comment:
maybe clearer: ... not allowed within the root of the schema, but only under a `properties` sub-schema

maybe add `(without them we can project a schema to a sub-path)`.